src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -581,6 +582,7 @@ class MultiDiracDeterminant : public WaveFunctionComponent
   std::shared_ptr<std::vector<int>> detData;
   std::shared_ptr<std::vector<std::pair<int, int>>> uniquePairs;
   std::shared_ptr<std::vector<RealType>> DetSigns;
+  std::shared_ptr<std::vector<int>> ndet_per_exc_lvl;
```

Comment:
Document this variable.

Reply:
I added a description (maybe it would also be helpful to add member function `int max_exc_lvl(){return ndet_per_exc_lvl.size()+1;}`?

Reply:
This is nice to have but kind of doubt it is absolutely needed. I expect use `ndet_per_exc_lvl` more directly.
Please also rename `exc_lvl`, use full words.

Reply:
`ndets_per_excited_level`

Reply:
fixed